### DISCUSSION TITLE
post/2020-07-03-use_prometheus/

### DISCUSSION THREAD

**Discussion by giscus** [zcong1993/blog#77] [Announcements]
# post/2020-07-03-use_prometheus/

生产环境的应用往往都需要 7x24 小时高强度服务用户, 上线之后如果想睡好觉, 就需要知晓应用运行的某些状态指标, 以此判断应用是否健康, 这也就是监控的意义之一. K8S 世界会有什么样的工具帮我们做好监控呢?

https://blog.cong.moe/post/2020-07-03-use_prometheus/

**Comment by wang-xiaowu** [comment]
您好请问咱们有计划编写配合 grafana 使用的文章么

**Reply by zcong1993** [reply]
我对 grafana 仅仅停留在会用的水平, 还是建议看相关书籍或者去抄开源项目的图表配置.

**Reply by wang-xiaowu** [reply]
好的，感谢，文章写的很好很有帮助
